client/components/_tests_/footer.test.tsx

```diff
@@ -0,0 +1,11 @@
+import renderer from "react-test-renderer";
+import Footer from "../footer";
+
+describe("footer test", () =>{
+    it('is redendring properly footer component ', () => {
+        const tree = renderer.create(
+            <Footer />
+        ).toJSON()
```

Comment:
why do you convert to json?

Reply:
to be more easy to read the test code

Reply:
because, as i remember when i don't convert them `toJSON` they stay not formatted